src/components/InvitationModal/index.tsx

```diff
@@ -0,0 +1,161 @@
+        setValidClassName('invalid-email');
+      }
+    });
+  }
+
+  return (
+    <div className="invitation-modal-content">
+      <h2>Share Schedule</h2>
+      <p>
+        Enter the email associated with another user&apos;s GT-Scheduler account
+        & we&apos;ll them an invite via email to import this schedule into their
+        view
+      </p>
+      <div className="email-block">
+        {/* <form>
+                    <text className='invalid-email'>Invalid Email</text>
+                    <input
+                        type="email"
+                        id="email"
+                        className="email"
+                        placeholder="recipient@example.com"
+                    />
+                </form> */}
+        <div className="email-input">
+          <input
+            type="email"
+            id="email"
+            value={input}
+            className="email"
+            placeholder="recipient@example.com"
+            list="recent-invites"
+            onChange={(e): void => setInput(e.target.value)}
+          />
+          <datalist id="recent-invites">
+            {emails.map((element) => (
+              <option>{element[0]}</option>
+            ))}
+          </datalist>
+          <text className={validClassName}>{validMessage}</text>
+        </div>
+        <button
+          type="button"
+          className="send-button"
+          onClick={(e): void => {
+            verifyUser();
+          }}
+        >
+          Send Invite
+        </button>
+      </div>
+      <hr className="divider" />
+      <p>
+        Users Invited to View <strong>Primary</strong>
+      </p>
+      <div className="shared-emails">
+        {emails.map((element) => (
+          <div className="email-and-status">
+            <div className="individual-shared-email" id={element[1]}>
+              {element[0]}
+              <Button className="button-remove">
+                <FontAwesomeIcon className="circle" icon={faCircle} />
+                <FontAwesomeIcon className="remove" icon={faClose} />
+              </Button>
+            </div>
+            <div className="status">
+              <div className="status-text">
+                <text>Status: {element[1]}</text>
+              </div>
+              <Button className="caret-down-button">
+                <FontAwesomeIcon className="caret-down" icon={faCaretDown} />
+              </Button>
+            </div>
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+}
+
+export type LoginModalProps = {
+  show: boolean;
+  onHide: () => void;
+};
+
+/**
+ * Component that can be used to show the invitaion modal.
+ */
+export default function InvitationModal({
+  show,
+  onHide,
+}: LoginModalProps): React.ReactElement {
+  return (
+    <Modal
+      show={show}
+      className="invatation-modal"
+      onHide={onHide}
+      buttons={[
+        { label: 'Cancel', onClick: (): void => onHide(), cancel: true },
+      ]}
```

Comment:
I feel like the modal is a little smaller than the design. Some styling also deviates slightly from the design, although this is very trivial. Make sure to compare the final modal with the design again :)